presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -334,6 +275,24 @@ Keytab files must be distributed to every node in the cluster that runs Presto.
 
 :ref:`Additional Information About Keytab Files.<hive-security-additional-keytab>`
 
+HDFS wire encryption
+--------------------
+
+In a Kerberized Hadoop cluster with enabled HDFS wire encryption you can enable
+Presto to access HDFS by using below property.
```

Comment:
```suggestion
Presto to access HDFS by using the ``hive.hdfs.wire-encryption.enabled`` property.
```
Below and above can be ambiguous. Suggest specifying the exact name so the sentence has meaning by itself, when possible.